shared/src/main/scala/squants/energy/MolarEnergy.scala

```diff
@@ -0,0 +1,48 @@
+package squants.energy
+
+import squants.mass.{ChemicalAmount, Moles}
+import squants.{AbstractQuantityNumeric, Dimension, PrimaryUnit, Quantity, SiUnit, UnitConverter, UnitOfMeasure}
+
+/**
+  *
+  * @author Nicolas Vinuesa
+  * @since 1.4
+  *
+  * @param value Double
```

Comment:
Its just a unit of energy over mass. The link could be: https://en.wikipedia.org/wiki/Joule_per_mole
Should I add it in the class? I've not seen it elsewhere...